packages/jetpack-logo/phpunit.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we have a base testing framework package? All packages would have it as a dev-dependency and would use it for building and running the package tests. Thoughts?